solidity/random-beacon/test/system/e2e.test.ts

```diff
@@ -37,15 +31,17 @@ const fixture = async () => {
 
   await registerOperators(
     contracts.randomBeacon as RandomBeacon,
-    (await getUnnamedAccounts()).slice(1, 1 + constants.groupSize)
+    contracts.t as T,
+    constants.groupSize,
+    1
```

Comment:
Can we add a comment this `1` is a result of how we use `ethers.getUnnamedSigners()` later?

Reply:
f931643f8c9330c677ed937b457aed7e45072fe9